src/context.rs

```diff
@@ -240,6 +241,7 @@ impl PySessionContext {
         ctx.register_udf(predict_xgb.clone());
         ctx.register_udf(ScalarUDF::from(GreatestFunc::new()));
         ctx.register_udf(ScalarUDF::from(LeastFunc::new()));
+        ctx.register_udf(ScalarUDF::from(HashIntFunc::new()));
```

Comment:
@ghoersti In here, we add the function to set of functions available in DataFusion, this is actually how all the functions are created in DataFusion via the ScalarUDF